test/test-segmentation-lopez-acosta-2019.jl

```diff
@@ -0,0 +1,28 @@
+using ImageSegmentation: segment_labels, segment_mean, labels_map
+
+@ntestset "$(@__FILE__)" begin
+    @ntestset "Lopez-Acosta 2019" begin
+        loadimg(s::String) = float64.(load(s))
+
+        truecolor = loadimg(
+            "./test_inputs/pipeline/input_pipeline/20220914.aqua.truecolor.250m.tiff"
+        )
+
+        falsecolor = loadimg(
+            "./test_inputs/pipeline/input_pipeline/20220914.aqua.reflectance.250m.tiff"
+        )
+
+        landmask = loadimg("./test_inputs/pipeline/input_pipeline/landmask.tiff")
+
+        segments = LopezAcosta2019()(truecolor, falsecolor, landmask)
+
+        @show segments
+        save(
+            "./test_outputs/segmentation-Lopez-Acosta-2019-mean-labels" *
+            Dates.format(Dates.now(), "yyyy-mm-dd-HHMMSS") *
+            ".png",
+            map(i -> segment_mean(segments, i), labels_map(segments)),
```

Comment:
Consider a more readable comprehension.

```suggestion
            [segment_mean(segments, i) for i in labels_map(segments)],
```

Reply:
This line is a direct copy from https://juliaimages.org/stable/pkgs/segmentation/ which I'd like to keep as-is, to help people who might be searching for it or asking an LLM about the code.

Reply:
@danielmwatkins and I settled on the map to keep consistency with the Julia Images library.